client/state/account-recovery/reset/test/reducer.js

```diff
@@ -250,4 +253,104 @@ describe( '#account-recovery/reset reducer', () => {
 
 		assert.isNull( state.validate.error );
 	} );
+
+	it( 'ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST action should set the requesting status flag as true.', () => {
+		const state = reducer( undefined, {
+			type: ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST,
+		} );
+
+		assert.isTrue( state.resetPassword.isRequesting );
+	} );
+
+	const requestingResetPasswordState = deepFreeze( {
+		resetPassword: {
+			isRequesting: true,
+		},
+	} );
+
+	it( 'ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS action should set the requesting flag as false.', () => {
+		const state = reducer( requestingResetPasswordState, {
+			type: ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS,
+		} );
+
+		assert.isFalse( state.resetPassword.isRequesting );
+	} );
+
+	it( 'ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR action should set the requesting flag as false.', () => {
+		const state = reducer( requestingResetPasswordState, {
+			type: ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR,
+			error: {},
+		} );
+
+		assert.isFalse( state.resetPassword.isRequesting );
+	} );
+
+	it( 'ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS action should set the succeeded flag as true.', () => {
+		const state = reducer( undefined, {
+			type: ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_SUCCESS,
+		} );
+
+		assert.isTrue( state.resetPassword.succeeded );
+	} );
+
+	const resetPasswordSucceededState = deepFreeze( {
+		resetPassword: {
+			succeeded: true,
+		},
+	} );
+
+	it( 'ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR action should set the succeeded flag as false.', () => {
+		const state = reducer( resetPasswordSucceededState, {
+			type: ACCOUNT_RECOVERY_RESET_PASSWORD_REQUEST_ERROR,
+			error: {},
+		} );
+
+		assert.isFalse( state.resetPassword.succeeded );
```

Comment:
The line 102 is `resetPassword.error` state, and this test is testing for the `resetPassword.succeeded` state, which is at [line 95](https://github.com/Automattic/wp-calypso/pull/13364/files#diff-b16e5d5c10f6c7c5b3b1a7eaa81e98a7R102). Thus this should be `false`. 🙂